tests/general/container_test.go

```diff
@@ -128,7 +128,8 @@ func TestConfigYamlEnvVar(t *testing.T) {
 exporters:
   otlp:
     endpoint: "${OTLP_ENDPOINT}"
-    insecure: true
+    tls:
```

Comment:
Do you mind keeping this and adding a comment saying it's intentional to ensure that converters apply to this config mechanism?

Reply:
Sure, I can open another PR undoing this change. I saw a few tests with the incorrect format on purpose that I left, but I didn't know this was also intentionally out-dated.